packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -26,18 +28,30 @@ abstract class Screen {
 class EmptyScreen extends Screen {
   const EmptyScreen(String name, this.icon) : super(name);
 
-  static const EmptyScreen inspector =
-      EmptyScreen('Flutter Inspector', Icons.map);
+  static final EmptyScreen debugger = EmptyScreen(
+    'Debugger',
+    Octicons.getIconData('bug'),
+  );
 
-  static const EmptyScreen timeline = EmptyScreen('Timeline', Icons.timeline);
+  static final EmptyScreen timeline = EmptyScreen(
+    'Timeline',
+    Octicons.getIconData('pulse'),
```

Comment:
I found a package with all the Octicons so I've switched these screens back to match the existing Flutter Web app. I have no strong preference towards these icons but they aren't horrible and are what users are expecting.

Reply:
That sounds reasonable. I would write the flutter_icons package a bit differently anyway if we were writing it from scratch so you can use the IconData in const contexts.
That said, it is a convenient way for us to get a wide range of good icons with the right license file so I'd recommend we just run with it until it comes time to roll the flutter app into g3.